dataland-internal-storage/src/test/resources/db/migration/V6/expectedTransformedDatabaseEntry.json

```diff
@@ -0,0 +1,121 @@
+{
+  "general": {
+    "fiscalYearDeviation": "Deviation",
+    "fiscalYearEnd": "2022-10-05",
+    "nfrdMandatory": null,
+    "numberOfEmployees": null,
+    "referencedReports": {
+      "SustainabilityReport": {
+        "reference": "50a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+        "reportDate": null,
+        "isGroupLevel": null,
+        "currency": "THB"
+      },
+      "ESEFReport": {
+        "reference": "50a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+        "reportDate": null,
+        "isGroupLevel": null,
+        "currency": "BSD"
+      }
+    },
+    "assurance": {
+      "assurance": "LimitedAssurance",
+      "provider": null,
+      "dataSource": {
+        "page": 91,
+        "report": "SustainabilityReport",
+        "tagName": "initiatives"
+      }
+    },
+    "scopeOfEntities": "NA",
+    "euTaxonomyActivityLevelReporting": "No",
+    "alignedActivities": [
+      {
+        "substantialContributionCriteria": {
+          "Water": 12
+        }
+      }
+    ]
+  },
+  "opex": {
+    "totalAmount": {
+      "comment": null,
+      "dataSource": null,
+      "value": {
+        "amount": 940716.1856535822,
+        "currency": null
+      },
+      "quality": "Estimated"
+    },
+    "nonEligibleShare": {},
+    "eligibleShare": {},
+    "substantialContributionToSustainableUseAndProtectionOfWaterAndMarineResources": 12,
+    "substantialContributionToClimateChangeMitigation": 5,
+    "substantialContributionToProtectionAndRestorationOfBiodiversityAndEcosystems": 7,
+    "alignedActivities": [
+      {
+        "other": 99,
```

Comment:
Do we have "other" in the this category?

Reply:
Has been discussed

Reply:
changed to activityName